framework/src/bundle/BundleResourceContainer.cpp

```diff
@@ -223,8 +223,6 @@ namespace cppmicroservices
         }
         catch (std::exception const& ex)
```

Comment:
`ex` can be removed since its not used. I'm surprised that the linux/macOs builds didn't fail with an "unused variable" warning.